### PR TITLE
Content additional fix validations

### PR DESCRIPTION
**Pull request by roysagi**
## Status
- [x] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
related: https://github.com/demisto/etc/issues/23712

## Description
some additional fixes to make content pass validation

### CODE REVIEW

File: Packs/Digital_Defense_FrontlineVM/Playbooks/playbook-Digital_Defense_FrontlineVM_-_PAN-OS_block_assets.yml [hunk 1]
```diff
@@ -247,7 +247,9 @@ tasks:
       id: 963529eb-2f34-4072-801f-9eed161669d1
       version: -1
       name: PAN-OS - Block IP and URL - External Dynamic List
-      description: ''
+      description: |-
+        This playbook blocks IP addresses and URLs using Palo Alto Networks Panorama or Firewall External Dynamic Lists.
```
Comment (ronykoz) [inline Packs/Digital_Defense_FrontlineVM/Playbooks/playbook-Digital_Defense_FrontlineVM_-_PAN-OS_block_assets.yml:251]: where did you take this from

Comment (roysagi) [reply Packs/Digital_Defense_FrontlineVM/Playbooks/playbook-Digital_Defense_FrontlineVM_-_PAN-OS_block_assets.yml:251]: when I downloaded it from demisto - it added this description. should I take it out?

Comment (ronykoz) [reply Packs/Digital_Defense_FrontlineVM/Playbooks/playbook-Digital_Defense_FrontlineVM_-_PAN-OS_block_assets.yml:251]: Make sure with @idovandijk @bakatzir

Comment (bakatzir) [inline Packs/Digital_Defense_FrontlineVM/Playbooks/playbook-Digital_Defense_FrontlineVM_-_PAN-OS_block_assets.yml:251]:
```suggestion
        This playbook blocks IP addresses and URLs using PAN-OS External Dynamic Lists.
```

**Comment by roysagi** [comment]
@ronykoz @yuvalbenshalom - this fails on thee marketplace stage in circle - needs force merge

**Pull request merged**